storage/mdcb_storage.go

```diff
@@ -256,3 +245,50 @@ func (m MdcbStorage) Exists(key string) (bool, error) {
 
 	return foundLocal && foundRpc, nil
 }
+
+// cacheCertificate saves locally resourceCertificate after pull from rpc
+func (m MdcbStorage) cacheCertificate(key, val string) error {
+	var err error
+	if m.CallbackOnPullCertificateFromRPC != nil {
+		err = (*m.CallbackOnPullCertificateFromRPC)(key, val)
+		if err != nil {
+			m.logger.WithError(err).Error("cannot save resourceCertificate locally")
+		}
+	}
+	return err
```

Comment:
```suggestion
	if m.CallbackOnPullCertificateFromRPC == nil {
	        return nil
	}
	
	return (*m.CallbackOnPullCertificateFromRPC)(key, val)
```
It's a little bit cleaner and we don't log the error twice (let's make sure that the function that calls `cacheCertificate` logs it. `cacheCertificate` could just return the error and it'd be enough. Wdyt?

Reply:
Way better!!

Reply:
I changed the pointer, is not longer a pointer as it doesnt makes sense